geomstats/geometry/spd_matrices.py

```diff
@@ -4,13 +4,14 @@
 
 import geomstats.backend as gs
 import geomstats.vectorization
+from geomstats.geometry.embedded_manifold import OpenSet
 from geomstats.geometry.general_linear import GeneralLinear
 from geomstats.geometry.matrices import Matrices
 from geomstats.geometry.riemannian_metric import RiemannianMetric
 from geomstats.geometry.symmetric_matrices import SymmetricMatrices
 
 
-class SPDMatrices(SymmetricMatrices):
+class SPDMatrices(SymmetricMatrices, OpenSet):
```

Comment:
Do we need SPDMatrices to inherit from SymmetricMatrices now? The corresponding methods could be accessed by `self.ambient_manifold.my_method`? Same question for the other manifolds in this case.

Reply:
The other manifolds don't inherit from the ambient manifold in general. the methods like expm, powerm are inherited here, which can be useful.. I don't know if you should keep this